src/_locales/en_US/messages.json

```diff
@@ -89,6 +89,12 @@
     "options_title": {
         "message": "Privacy Badger Options"
     },
+    "options_dnt_policy_setting": {
+        "message": "Check if sites have a DNT policy and unblock them"
```

Comment:
I dunno, https://www.eff.org/dnt-policy links there already. We might add some more explanatory text to this checkbox in the future too, but no need to wait on that I think.